etl/steps/data/garden/worldbank_wdi/2023-05-29/wdi.meta.yml

```diff
@@ -2631,27 +2631,45 @@ tables:
         title: GDP per capita (current US$)
         short_unit: $
         unit: current US$
+        presentation:
+          topic_tags:
+            - "Economic Growth"
```

Comment:
I think you don't need the `"` or the indentation, you could simply pass:
```
topic_tags:
- Economic Growth
```
But if this works well, then all good.